app/services/ClickHandler.coffee

```diff
@@ -59,7 +59,8 @@ class ClickHandler
           @board.deleteCells @tuplesClicked()
           @clicked = []
           if @BoardSolvedService.isCleared @board.boardValues[@board.dimension - 1]
-            setTimeout (() => @BoardSolvedService.createNewBoard()), 100
+            @board.successAnimation()
+            setTimeout (() => @BoardSolvedService.createNewBoard()), 500
```

Comment:
I can potentially see it being a problem where someone presses the `New Game` button while in the success animation or even before the timeout finishes. This would cause `New Game` to be effectively pressed twice.